bin/supervisor/src/flags/supervisor.rs

```diff
@@ -0,0 +1,185 @@
+use clap::Args;
+use std::net::IpAddr;
+
+/// Supervisor configuration arguments.
+#[derive(Args, Debug)]
+pub struct SupervisorArgs {
+    /// L1 RPC source.
+    #[arg(long, env = "L1_RPC", help = "L1 RPC source")]
```

Comment:
there is some way you can get clap to use the rustdocs as help, not sure if that is what happens by default. I know if you have some escape chars in there you should use `verbatim_doc_comment` attribute to get it to print unescaped as help

feel free to leave the doc without trailing punctuation for sake of simplicity if that's why the help is now double up. is safer that way since means we don't have to maintain the invariant docs = help message manually.

Reply:
Updated.